src/doc/trpl/ownership.md

```diff
@@ -106,8 +106,8 @@ take(v);
 println!("v[0] is: {}", v[0]);
 ```
 
-Same error: “use of moved value.” When we transfer ownership to something else,
-we say that we’ve ‘moved’ the thing we refer to. You don’t need some sort of
+Same error: “use of moved value”. When we transfer ownership to something else,
```

Comment:
This should actually be single curlies rather than double, would you mind fixing that?